ubi_manifest/worker/tasks/depsolver/rpm_depsolver.py

```diff
@@ -49,11 +49,23 @@ def __init__(
 
         self._srpm_repos: list[Future[YumRepository]] = srpm_repos
 
-        self._provides: set[RpmDependency] = set()  # set of rpm.provides we've visited
-        self._requires: set[RpmDependency] = set()  # set of rpm.requires we've visited
-
-        # set of solvables (pkg, lib, ...) that we use for checking remaining requires
-        self._unsolved: set[RpmDependency] = set()
+        self._provided_rpms: set[RpmDependency] = (
+            set()
+        )  # set of rpm.provides we've visited
+        self._required_rpms: set[RpmDependency] = (
+            set()
+        )  # set of rpm.requires we've visited
+
+        self._required_files: set[RpmDependency] = (
```

Comment:
I guess you can use only `set[str]` for files but I guess it's acceptable with `set[RpmDependency]`

Reply:
I played around with that but decided on `RpmDependency` so it can share `what_provides`